components/contribution-flow/ContributionFlowStepsProgress.js

```diff
@@ -30,7 +30,7 @@ const StepLabel = styled(Span)`
 const PrettyAmountFromStepDetails = ({ stepDetails, currency, isFreeTier, isCrypto }) => {
   if (stepDetails.amount) {
     const totalAmount = stepDetails.amount + (stepDetails.platformContribution || 0);
-    return (
+    return !isCrypto ? (
```

Comment:
We corrected this in past PR

Reply:
Yes, my mistake, I missed this in resolving merge conflicts, I've done the conflict resolution again and this time it's good I think. 👍🏽